### PR TITLE
 Update deps to fix vulnerability

### PR DESCRIPTION
**Pull request by rkostyuchenko**
This PR updates deps to fix CVE:

-  [minimatch ReDoS vulnerability ](https://github.com/advisories/GHSA-f8q6-p94x-37v3)

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Migration
 
+## 4.1.0 → 5.0.0
```
Comment (igoradamenko) [inline MIGRATION.md:3]: Please, add a note to CHANGELOG about the new version too.

**Review by igoradamenko — CHANGES_REQUESTED** [review]

File: MIGRATION.md (hunk 1 above)
Comment (rkostyuchenko) [reply MIGRATION.md:3]: [Done](https://github.com/funbox/frontend-tests-runner/pull/20/commits/ff9f884390e7ee392bbc5a3f1f7fb42cf1b11883)

Comment (igoradamenko) [reply MIGRATION.md:3]: Oh, wait. But the version of the package has not been changed.

Do you mind to bump it too? Set `version: 5.0.0` in package.json & package-lock.json.

**Review by igoradamenko — CHANGES_REQUESTED** [review]

File: MIGRATION.md (hunk 1 above)
Comment (rkostyuchenko) [reply MIGRATION.md:3]: Thanks, fixed

**Review by igoradamenko — APPROVED** [review]

**Comment by igoradamenko** [comment]
Published @funboxteam/frontend-tests-runner@5.0.0.

Thanks 🤝

**Pull request merged**